proto/zetachain/zetacore/crosschain/tx.proto

```diff
@@ -223,6 +223,10 @@ message MsgVoteInbound {
 
   // confirmation mode used for the inbound
   ConfirmationMode confirmation_mode = 21;
+
+  // error_message carries information about the error that caused non-SUCCESS
+  // status of inbound observation
+  string error_message = 22;
```

Comment:
Do we need this error message field to be part of the msg.Digest() ?
Imo it's safer to just remove it if not needed , we have the status as part of the digest already.

Reply:
I think this is a good point. The observation status in the `MsgVoteInbound` already indicates the category of the failure. The only side effect is that the final `errorMessage` is determined by the finalizing vote ( the 6th vote).

Reply:
Yeah, the message itself is just for debugging, so it might not be a problem if we use it for the final message 
I can't also see any problems with keeping it in the digest, since we directly abort the cctx anyway. Up to you either seems fine

Reply:
Even when the cctx is aborted/reverted the risk of double observation remain.

Reply:
Excluded it from digest https://github.com/zeta-chain/node/pull/4211/commits/dbb0bbaf895ac991ed6b560eea97564ddea0a978